pkg/cmd/roachtest/tests/large_schema_backup_restore.go

```diff
@@ -0,0 +1,159 @@
+// Copyright 2024 The Cockroach Authors.
+//
+// Use of this software is governed by the CockroachDB Software License
+// included in the /LICENSE file.
+
+package tests
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster"
+	"github.com/cockroachdb/cockroach/pkg/cmd/roachtest/option"
+	"github.com/cockroachdb/cockroach/pkg/cmd/roachtest/registry"
+	"github.com/cockroachdb/cockroach/pkg/cmd/roachtest/spec"
+	"github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test"
+	"github.com/cockroachdb/cockroach/pkg/roachprod/install"
+	"github.com/stretchr/testify/require"
+)
+
+// registerLargeSchemaBackupRestores registers all permutations of
+// multi-region large schema benchmarking (for different scales
+// and multi-region).
+func registerLargeSchemaBackupRestores(r registry.Registry) {
+	// 10k is probably the upper limit right now, but we haven't tested further.
+	for _, scale := range []int{1000, 5000, 10000} {
+		largeSchemaBackupRestore(r, scale)
+	}
+}
+
+func largeSchemaBackupRestore(r registry.Registry, numTables int) {
+	clusterSpec := []spec.Option{
+		spec.CPU(8),
+		spec.WorkloadNode(),
+		spec.WorkloadNodeCPU(8),
+		spec.VolumeSize(800),
+		spec.GCEVolumeType("pd-ssd"),
```

Comment:
how big is this cluster once set up ? we need pd ssds, or can we get use cheap local ssds (375 GB per node)